source/renderer/app/components/voting/widgets/PatchCheckbox.js

```diff
@@ -0,0 +1,27 @@
+// @flow
+import React from 'react';
+import type { ComponentType, Element } from 'react';
+
+import { Checkbox } from 'react-polymorph/lib/components/Checkbox';
+
```

Comment:
@lucas-barros please remove this blank line 🙏

Reply:
Updated, thanks!